samples/tei/00000010.xml

```diff
@@ -0,0 +1,861 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<TEI.2><teiHeader/><text><body>
```

Comment:
This is some seriously old TEI? Havent seen `<TEI.2>` element in forever. Is that even part of TEI P5?

Reply:
1) Yes, the original source is AFAIK digizeitschriften. 2) I don't know.

Are there anywhere else any freely available examples of similar (but maybe more recent) TEI files?

Reply:
@cneud wrznr @bertsky @tboenig Any ideas for sensible samples? Would also help in the efforts for file format transformation in the [DHd AG OCR](https://dhd-ag-ocr.github.io).